packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -188,14 +188,34 @@ class ContainerPlugin {
           library,
         },
         (error: WebpackError | null | undefined) => {
-          if (error) {
-            return callback(error);
+          if (error) return callback(error);
+          if (compilation.options.optimization.runtimeChunk) {
```

Comment:
is there other way or better way to check for runtime chunk?

Reply:
@2heal1 @zhoushaw do you have any feedback on this implementation? 

This is to ensure that all runtime chunks contain critical runtimeModules for federation, so that container/entery/xxx can be used against hosts "webpack-runtime.js" as well as "remoteEntry.js"

Reply:
The alternative here is use 2x containerPlugins, but this causes runtime requirement and runtime modules like share module or initialize runtime module to be duplicated as well, when all i want is the dependency assigned to 2 runtime chunks, not re-apply the entire container plugin again to the same build.